spaces/domains.json

```
@@ -828,4 +828,5 @@
   "vote.wandinvestments.com": "wandinvestments.com",
   "vote.lebo.finance": "lebo-finance.eth",
   "vote.cirip.io": "cirip.io"
```

A comma is missing at the end of the line of your sub-domain.
```suggestion
  "vote.cirip.io": "cirip.io",
```